addons/mail/static/src/models/composer/composer.js

```diff
@@ -824,8 +824,11 @@ function factory(dependencies) {
                     const escapedSource = String(source).replace(
                         /([.*+?=^!:${}()|[\]/\\])/g,
                         '\\$1');
+                    const escapedEntitiesSource = String(_.escape(source)).replace(
+                        /([.*+?=^!:${}()|[\]/\\])/g,
+                        '\\$1');
                     const regexp = new RegExp(
-                        '(\\s|^)(' + escapedSource + ')(?=\\s|$)',
+                        '(\\s|^)(' + escapedSource + '|' + escapedEntitiesSource + ')(?=\\s|$)',
```

Comment:
I don't understand this change, could you please explain it?
It looks unrelated to opw and also it looks like no test is covering it?

Reply:
This is to do the emoji check with and without the escaping. Because otherwise in the links or in the document icons such as `:'(` have an escape and therefore are no longer displayed as an icon.